dali/test/python/test_dali_variable_batch_size.py

```diff
@@ -289,6 +364,8 @@ def test_float_array_ops():
     (fn.noise.gaussian, {}),
     (fn.noise.shot, {}),
     (fn.noise.salt_and_pepper, {'devices': ['cpu']}),  # TODO(janton): Enalble GPU once salt_and_pepper supports it.
+    (fn.segmentation.random_mask_pixel, {'devices': ['cpu']}),
+    (fn.roi_random_crop, {'crop_shape': [10, 15, 3], 'roi_start': [25, 20, 0], 'roi_shape': [40, 30, 3], 'devices': ['cpu']})
```

Comment:
Same nitpick as above. Feel free to ignore it if you think otherwise

Reply:
Done